scripts/update-browser-releases/edge.ts

```diff
@@ -9,6 +9,69 @@ import stringify from '../lib/stringify-and-order-properties.js';
 
 import { newBrowserEntry, updateBrowserEntry } from './utils.js';
 
+//
+// Content of the two release note files
+//
+let releaseNotesText;
+let archivedReleaseNotesText;
+
+/**
+ * initReleaseNoteFiles - Fetch both release notes file and store them
+ */
+const initReleaseNoteFiles = async () => {
+  // Fetch the regular page
+  const releaseNote = await fetch(
+    'https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel',
+  );
+
+  // Check that it exists, if not exit.
+  if (releaseNote.status != 200) {
+    // File not found -> log a warning
+    console.log(
+      chalk`{red \nRelease note files not found for Edge (${releaseNote.status}).`,
```

Comment:
Wouldn't it be better to throw an error here and refrain from updating Edge if the resource couldn't be loaded?  Assuming that we're not already handling this case.

Reply:
Most of the data comes from the API (updateedge); only the URL of the release note comes from this file.

If the release note file is not found, all other fields are updated but `release_note`.

The person reviewing the PR generated by the workflow will see this message and will have to "manually" update this field (and likely try to understand why): the script will have performed as much as possible without human intervention.

Reply:
Ah, okay!  In that case, LGTM!